grails-app/views/project/list.gsp

```diff
@@ -48,8 +48,14 @@
                         <div class="col-sm-6">
                             <div class="card-filter">
                                 <div class="btn-group pull-right" role="group" aria-label="...">
-                                    <a href="${createLink(action:'list', params:[mode:'grid'])}" class="btn btn-default btn-xs ${params.mode != 'grid' ? '' : 'active'}"><i class="glyphicon glyphicon-th-large "></i></a>
-                                    <a href="${createLink(action:'list')}" class="btn btn-default btn-xs ${params.mode == 'grid' ? '' : 'active'}"><i class="glyphicon glyphicon-th-list"></i></a>
+                                    <a href="list?${queryStringParams}mode=grid"
```

Comment:
I would prefer `createLink` to be left in here.  To achieve that you'd just need to build queryStringParams as a map instead of a string and do `params: queryStringParamsMap + [mode:'grid']`.

Reply:
Thanks, I struggled with getting it to work on the custom landing page but I've worked out a solution.